pkgs/record_use/README.md

```diff
@@ -0,0 +1,116 @@
+# `package:record_use`
+
+
+> [!CAUTION]
+> This is an experimental package, and it's API can break at any time. Use at
+> your own discretion.
+
+This package provides the data classes for the usage recording feature in the
+Dart SDK.
+
```

Comment:
You might want to say something like 'this package is the canonical way to read usage files written by the sdk' (`RecordedUsages.fromFile`); if this is the main way to read those files, that could give us a bit more flexibility in terms of evolving the format.

Reply:
This is the only way to read the files!